manifests/0000_51_olm_06_deployment.yaml

```diff
@@ -45,6 +45,8 @@ spec:
           value: quay.io/openshift/origin-olm-catalogd:latest
         - name: OPERATOR_CONTROLLER_IMAGE
           value: quay.io/openshift/origin-olm-operator-controller:latest
+        - name: RUKPAK_IMAGE
+          value: quay.io/openshift/origin-olm-rukpak:latest
```

Comment:
We also need to add an image reference for this image in this file: https://github.com/openshift/cluster-olm-operator/blob/main/manifests/image-references.pending

Reply:
I have that in my PR #16. We can discuss the order and split up